ROS/controls/README.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Perhaps some thing to include in the readme are file architecture, general overview of what the code does in the grander scheme of the project, etc.